profiles/abhinavkashyap061.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/84775901/170862408-0209e8c2-e304-43d1-becc-223e72bfa9e6.png",
+    "issueId": 1142,
+    "name": "Abhinav Kashyap",
+    "username": "abhinavkashyap061"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```